canonical-web-and-design/ubuntu.com
# Copy update on /iot/digital-signage

## Done

- Copy update on /iot/digital-signage

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Be sure to test on mobile, tablet and desktop screen sizes
- Please check the page against [copy doc](https://docs.google.com/document/d/1fFvBl-ZZvKDFV6OMv7O4kudQKtyxfvSU60rfVdKluAQ/edit#heading=h.a5ulsi211j3d) and resolve comments

## Issue / Card

Fixes [#4723](https://github.com/canonical-web-and-design/web-squad/issues/4723)


## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
